bernabe_chavajay/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <title>Google</title>
+    <link rel="stylesheet" href="style.css">
+
+    
+</head>
+<body>
+    <div class="navbar">
+        <button class="settings">Google</button>
+
+        <a href="https://mail.google.com">Gmail</a>
+
+        <a href="https://www.google.com.gt/imghp?hl=es&authuser=0&ogbl">Imagenes</a>
+    </div>
+
+    <div class="logo">
+        <img src="logo.png" alt="logo-google"
```

Comment:
Please use the image from the url and not include the image on the repository

Reply:
Got it

Reply:
Waiting for this change

Reply:
I did the change